src/script/lua_api/l_env.cpp

```diff
@@ -340,6 +340,38 @@ int ModApiEnv::l_swap_node(lua_State *L)
 	return 1;
 }
 
+// bulk_swap_node([pos1, pos2, ...], node)
+// pos = {x=num, y=num, z=num}
+int ModApiEnv::l_bulk_swap_node(lua_State *L)
+{
+	GET_ENV_PTR;
+
+	// parameters
+	if (!lua_istable(L, 1)) {
+		return 0;
+	}
```

Comment:
should be `luaL_checktype`. we shouldn't be loose with types for new functions

Reply:
luaL_checktype throws an error if the type is wrong.
the old code silently does nothing if the type is wrong, this is bad practice